.github/workflows/pull-request-management.yml

```diff
@@ -87,7 +82,15 @@ jobs:
     if: needs.file-changes.outputs.eos_design == 'true' || needs.file-changes.outputs.config_gen == 'true' || needs.file-changes.outputs.requirements == 'true'
     steps:
       - uses: actions/checkout@v2
-      - uses: pre-commit/action@v2.0.2
+      - name: Install requirements
+        run: |
+          pip install -r ansible_collections/arista/avd/requirements-dev.txt
+          pip install -r ansible_collections/arista/avd/requirements.txt
+      - name: Run pre-commit
+        run: |
+          git branch -a
```

Comment:
Is this command important for the CI ? I would think it is more for debug

Reply:
correct, we should remove it